go.sum

```diff
@@ -21,18 +21,21 @@ github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRF
 github.com/apache/thrift v0.12.0/go.mod h1:cp2SuWMxlEZw2r+iP2GNCdIi4C1qmUzdZFSVb+bacwQ=
```

Comment:
let's move the go.sum update to another PR, I don't think UI changes need to check in go.sum

Reply:
To be honest, I'm not sure why `go.sum` has changed.

I'll remove this file from this pr